addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -990,3 +998,11 @@ section, .oe_img_bg, [data-oe-shape-data] {
         margin-bottom: 1rem;
     }
 }
+
+code.o_inline_code {
+    padding: map-get($spacers, 1);
+    border: $border-width solid $border-color;
+    border-radius: $border-radius;
+    background-color: $gray-100;
+    line-height: 2;
```

Comment:
I think that this is missing a `color: $gray-900;`  😉

----
![image](https://github.com/odoo/odoo/assets/8558351/d618509e-0265-4a1f-bf43-6a882d6b190c)